cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicationPolicyManager.java

```diff
@@ -20,16 +20,31 @@
 import java.util.Arrays;
 import java.util.List;
 
+/**
+ * Class to manage replication lifecycle events.
+ */
 public class ReplicationPolicyManager {
 
     private final List<Replicator> replicators = new ArrayList<Replicator>();
     private final ReplicationListener replicationListener = new ReplicationListener();
 
+    /**
+     * Interface with callback methods for replication lifecycle events.
```

Comment:
@ricellis Android does use `ReplicationPolicyManager`, but it's hidden from the user (it's all done under the hood in ReplicationService), whereas on Java SE you'd use it directly. I guess it might make sense to put the listener in it's own class as it perhaps could be confusing as it is.